### PR TITLE
Tests for cloud app

### PR DESCRIPTION
**Pull request by jhellar**
Actual tests that verify that data are in correct mongo.

### CODE REVIEW

**Comment by jhellar** [comment]
@flashlee can you take a look?

File: tests/cloudapp.js [hunk 1]
```diff
@@ -35,78 +43,134 @@ describe('Cloud App', function() {
...
+    console.log('connect to mongo');
+    const userUrl = `mongodb://admin:${userAdminPassword}@localhost:27017`;
     userClient = await MongoClient.connect(userUrl);
+    const systemUrl = `mongodb://admin:${systemAdminPassword}@localhost:27018`;
+    systemClient = await MongoClient.connect(systemUrl);
 
-    // TODO: get cloud app pod name
-
+    // get cloud app id
+    cloudAppId = cloudAppUrl.substring(cloudAppUrl.indexOf('//') + 2).split('.')[0].split('-')[1];
```
Comment (flashlee) [inline tests/cloudapp.js:105]: @jhellar can't be cloudAppId fetched from results of commands importAppZip or appDeploy?

**Review by flashlee — COMMENTED** [review]
LGTM. Only using timeouts is not good, but if there is no other solution I'm ok with that.
Also need to check if there is no other way to fetch cloudAppId

File: tests/cloudapp.js (hunk 1 above)
Comment (jhellar) [reply tests/cloudapp.js:105]: @flashlee I also tried to get this id with fhc, but didn't find it

**Pull request merged**